### PR TITLE
Feat add avatar component

### PR DESCRIPTION
**Pull request by nerdyspook**
**Added Avatar Component**
> Profile Avatar
> Sizes Avatar
> Square Avatar

avatar

### CODE REVIEW

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.avatar {
+    height: var(--sp-4);
+    width: var(--sp-4);
+    border-radius: 50%;
```
Comment (mansi2011) [inline components/avatar/avatar.css:4]: you can convert % into rem here as well

**Review by tanishk15sharma — APPROVED** [review]
clean code

**Comment by mansi2011** [comment]
great work and clean code

File: components/avatar/avatar.css (hunk 1 above)
Comment (nerdyspook) [reply components/avatar/avatar.css:4]: Thanks, it's just my preference to use % instead of rem while making a circle.

**Pull request merged**